src/pages/api/fetch-datasets.ts

```diff
@@ -0,0 +1,14 @@
+import type {NextApiRequest, NextApiResponse} from 'next';
```

Comment:
Or can we call the dataset fetcher client-side, without this API endpoint and without React Query?

Reply:
I've given my own question some more thought :)

Yes, it is possible to make the dataset fetcher (also) work client-side. This can be elegant: we can then use the same fetcher on the server and the client - the beauty of JavaScript. But I'm not sure yet about the requirements for the dataset fetcher. For example, we currently do not have the use API keys to query the data store. But this could change. If so, we'll have to find a way to work with these keys on the client as well. So for now I'd propose to continue with your current implementation, with React Query and an API endpoint.

Reply:
Good question, it was also something I was wondering. The fetcher now uses packages that only work for NodeJs. Because I was also not sure if secrets needed to be added later, like API keys, I made the decision to always do the calls server side. Instead of changing the fetcher to work client side as well.

Yes, let's keep it like this for now, but keep this question in mind for the future.